projects.md

```diff
@@ -0,0 +1,67 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation,
+all members may pursue leads as they come up.
+These may be a client or a grant opportunity.
+Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
+
+If a member decides to pursue a **lead**,
+they can reach out for preliminary discussions to see whether there is a potential to work together.
+If the lead expresses intention to proceed with a concrete project in mind,
+this becomes a **prospect**.
+The member now leads this sales effort and should draft an **internal proposal** and present it at a meeting to see whether the co-operative wants to move forward to submit a formal proposal to the prospective client.
```

Comment:
To confirm, they lead the sales effort while maybe not having access to CRM? Does this feel ok?